### PR TITLE
codewhisperer: add  logs to supplemental context feature

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andrewyuq — APPROVED** [review]

File: src/codewhisperer/util/editorContext.ts [hunk 1]
```diff
@@ -219,15 +219,15 @@ function logSupplementalContext(supplementalContext: CodeWhispererSupplementalCo
...
+    getLogger().debug(`CodeWhispererSupplementalContext:
             isUtg: ${supplementalContext.isUtg},
             isProcessTimeout: ${supplementalContext.isProcessTimeout},
             contentsLength: ${supplementalContext.contentsLength},
             latency: ${supplementalContext.latency},
         `)
 
     supplementalContext.supplementalContextItems.forEach((context, index) => {
-        getLogger().verbose(`
+        getLogger().debug(`
```
Comment (justinmk3) [inline src/codewhisperer/util/editorContext.ts:230]: Logging related messages in separate logger calls should not be done in a loop. Build _one_ string, then log that single string.

Comment (Will-ShaoHua) [reply src/codewhisperer/util/editorContext.ts:230]: understood

File: src/codewhisperer/util/editorContext.ts [hunk 2]
```diff
@@ -219,20 +219,21 @@ function logSupplementalContext(supplementalContext: CodeWhispererSupplementalCo
...
-                -----------------------------------------------
-            `)
+        logString += `Chunk ${index}:
+        Path: ${context.filePath}
+        Content: ${index}:${context.content}
+        Score: ${context.score}
+        -----------------------------------------------`
     })
+
+    getLogger().debug(logString)
```
Comment (justinmk3) [inline src/codewhisperer/util/editorContext.ts:238]: This is an improvement, thanks. But there is still a disconnected `getLogger().debug()` call on line 222. Only one one message is needed, not multiple disconnected ones. That also has the advantage of putting a single timestamp on the message.

Comment (Will-ShaoHua) [reply src/codewhisperer/util/editorContext.ts:238]: yea make sense, let me update it

Comment (Will-ShaoHua) [reply src/codewhisperer/util/editorContext.ts:238]: knowledge learned thanks Justin

File: src/codewhisperer/util/editorContext.ts [hunk 3]
```diff
@@ -219,20 +219,21 @@ function logSupplementalContext(supplementalContext: CodeWhispererSupplementalCo
...
-    getLogger().verbose(`
-            isUtg: ${supplementalContext.isUtg},
-            isProcessTimeout: ${supplementalContext.isProcessTimeout},
-            contentsLength: ${supplementalContext.contentsLength},
-            latency: ${supplementalContext.latency},
-        `)
-
+    let logString = ''
+
+    logString += `CodeWhispererSupplementalContext:
```
Comment (justinmk3) [inline src/codewhisperer/util/editorContext.ts:224]:
```suggestion
    let logString = `CodeWhispererSupplementalContext:
```

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**